core/src/main/java/org/jacorb/orb/giop/ClientConnectionManager.java

```diff
@@ -143,9 +143,9 @@ public synchronized ClientConnection getConnection(org.omg.ETF.Profile profile)
         }
         else
         {
-            if( logger.isInfoEnabled())
+            if( logger.isDebugEnabled())
             {
-                logger.info("ClientConnectionManager: found "
+                logger.debug("ClientConnectionManager: found "
```

Comment:
We could strip out the 'if block' if parameterized logger messages were used e.g.
```
logger.debug("ClientConnectionManager: found {}", clientConnection.getGIOPConnection())
```

Reply:
made the changes as suggested.